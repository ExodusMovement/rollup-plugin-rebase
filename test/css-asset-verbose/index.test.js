import { bundle, clean, list, read } from "../util"

const root = __dirname

test("CSS Asset Verbose", async () => {
  console.log = jest.fn()
  await bundle(root, "index.js", "output/index.js", { verbose: true })
  expect(console.log.mock.calls).toMatchSnapshot()

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/gToHHcDx.css")).toMatchSnapshot()

  await clean(root, "output")
})
