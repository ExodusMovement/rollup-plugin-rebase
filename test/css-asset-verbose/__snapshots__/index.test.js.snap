// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSS Asset Verbose 1`] = `
Array [
  Array [
    "Copying /Users/swerner/Workspace/open-source/rollup-plugin-rebase/test/css-asset-verbose/content.md => /Users/swerner/Workspace/open-source/rollup-plugin-rebase/test/css-asset-verbose/output/bsdudxaF.md...",
  ],
  Array [
    "Processing /Users/swerner/Workspace/open-source/rollup-plugin-rebase/test/css-asset-verbose/index.css => /Users/swerner/Workspace/open-source/rollup-plugin-rebase/test/css-asset-verbose/output/gToHHcDx.css...",
  ],
]
`;

exports[`CSS Asset Verbose 2`] = `
"bXipeBrs.png
bsdudxaF.md
gToHHcDx.css
gToHHcDx.css.map
index.js"
`;

exports[`CSS Asset Verbose 3`] = `
"import content from './bsdudxaF.md';
import css from './gToHHcDx.css';

function index() {
  return content + css
}

export default index;
"
`;

exports[`CSS Asset Verbose 4`] = `
"body {
  font-size: 20px;
  background: url(./bXipeBrs.png) top left repeat;
}

/*# sourceMappingURL=gToHHcDx.css.map */"
`;
